commit 70097af27385be715248e04a8a8952a82b922ba3
Author:     Dan Davison <dandavison7@gmail.com>
AuthorDate: Mon Jun 24 10:34:30 2019 -0400
Commit:     Dan Davison <dandavison7@gmail.com>
CommitDate: Mon Jun 24 10:58:19 2019 -0400

    Use target file to determine extension
    
    TODO: should use the unique extension obtained from both

diff --git a/src/main.rs b/src/main.rs
index f9c3058..ae07569 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -21,7 +21,8 @@ fn main() {
     let mut patch_set = PatchSet::new();
     patch_set.parse(&mut input).ok().expect("Error parsing input as a diff");
     for patched_file in patch_set {
-        let path = Path::new(&patched_file.source_file);
+        // TODO: use both source and target to determine language
+        let path = Path::new(&patched_file.target_file);
         let extension = path.extension()
             .expect(format!("Error determining file type: {}", path.to_str().unwrap()).as_str());
         let extension_str = extension.to_str().unwrap();
